cellprofiler/modules/makeprojection.py

```diff
@@ -64,73 +66,75 @@ class MakeProjection(cpm.Module):
+       sinusoidally with depth. The frequency is measured in z-stack steps
+       and pixels that vary with the given frequency will have a higher
+       score than other pixels with similar variance, but different
+       frequencies.
+    -  *%(P_BRIGHTFIELD)s:* Perform the brightfield projection at each
+       pixel position.
+       Artifacts such as dust appear as black spots which are most strongly
+       resolved at their focal plane with gradually increasing signals
+       below. The brightfield method scores these as zero since the dark
+       appears in the early z-stacks. These pixels have a high score for the
+       variance method but have a reduced score when using the brightfield
+       method.
+    -  *%(P_MASK)s:* Compute a binary image of the pixels that are masked
+       in any of the input images.
+       The mask method operates on any masks that might have been applied to
+       the images in a group. The output is a binary image where the “1”
+       pixels are those that are not masked in all of the images and the “0”
+       pixels are those that are masked in one or more of the images.
+       You can use the output of the mask method to mask or crop all of the
+       images in a group similarly. Use the mask method to combine all of
+       the masks in a group, save the image and then use **Crop**,
+       **MaskImage** or **MaskObjects** in another pipeline to mask all
+       images or objects in the group similarly.
+
+**References**
+    -  Selinummi J, Ruusuvuori P, Podolsky I, Ozinsky A, Gold E, et al.
+       (2009) “Bright field microscopy as an alternative to whole cell
+       fluorescence in automated analysis of macrophage images”, *PLoS ONE*
+       4(10): e7497 `(link)`_.
+
+.. _(link): http://dx.doi.org/10.1371/journal.pone.0007497
+                    """% globals())
 
         self.projection_image_name = cps.ImageNameProvider(
                 'Name the output image',
-                'ProjectionBlue', doc='''
-            Enter the name for the projected image.''',
+                'ProjectionBlue', doc="Enter the name for the projected image.",
                 provided_attributes={cps.AGGREGATE_IMAGE_ATTRIBUTE: True,
                                      cps.AVAILABLE_ON_LAST_ATTRIBUTE: True})
         self.frequency = cps.Float(
-                "Frequency", 6.0, minval=1.0, doc="""
-            <i>(Used only if %(P_POWER)s is selected as the projection method)</i><br>
-            This setting controls the frequency at which the power
-            is measured. A frequency of 2 will respond most strongly to
-            pixels that alternate between dark and light in successive
-            z-stack slices. A frequency of N will respond most strongly
-            to pixels whose brightness cycle every N slices.""" % globals())
+                "Frequency", 6.0, minval=1.0, doc="""\
+*(Used only if %(P_POWER)s is selected as the projection method)*
```

Comment:
Please put an empty line between the *(Used only...)* line and the next one.

Reply:
Add quotes around setting text: `"%(P_POWER)s"`